trymito.io/styles/[slug].module.css

```diff
@@ -1,19 +1,86 @@
+.decorative_line {
+    position: absolute;
+    width: 1px;
+    height: calc(100% - 42rem);
+    border: 0.5px dashed rgba(208, 208, 208, 0.25);
+    left: max(19.5rem, calc(((100vw - 70rem) / 2) - 6px));
+}
 
+/* Post Styling */
 .post {
-    margin-top: 4rem;
+    margin-top: 7rem;
     margin-bottom: 4rem;
+    width: 100%;
+    font-family: Inter, Ubuntu, Helvetica, Arial, sans-serif;
+}
+
+.post a, .suggested_post h4 {
+    text-decoration: underline 0.15em rgba(0, 0, 0, 0);
+    transition: text-decoration-color 300ms;
+}
+  
+.post a:hover, .suggested_post:hover h4 {
+    text-decoration-color: rgba(0, 0, 0, 1);
+}
+
+.post strong {
+    color: var(--color-text-primary);
+    font-weight: 500;
+}
+
+
+
+/* Post Banner */
+.post_banner {
+    background-color: var(--color-purple);
+    padding: 5rem 22rem;
+    width: 100%;
+    display: flex;
+    justify-content: center;
 }
 
 .post_title {
-    max-width: min(800px, 90vw);
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
 }
 
 .post_title h1 {
     font-size: 3rem;
+    max-width: 70rem;
+}
+
+.post_title p, .post_title h1 {
+    margin-top: 0
 }
 
+.post_metadata {
```

Comment:
This needs a bit of love on mobile. Maybe on mobile we can just show the date left aligned with the title and call it a day

<img width="202" alt="Screenshot 2024-02-19 at 10 33 27 AM" src="https://github.com/mito-ds/mito/assets/18709905/9a936707-8a74-44fb-94e1-0102ebdb4328">

Reply:
How does this look? 
![image](https://github.com/mito-ds/mito/assets/6673460/38b4fc91-23a1-432c-a5db-62f861edf59f)

Reply:
Ooh I think this looks good for the meta data, but can we just left align all of the set so that its not floating in the middle?